### PR TITLE
add prometheus basics tutorial

### PR DESCRIPTION
**Pull request by yolossn**
Disclaimer: I am the author of that article.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Prometheus is an open-source systems monitoring and alerting toolkit.
 - [Monitor your GitHub Repos with Docker and Prometheus](https://www.brianchristner.io/monitor-your-github-repos-with-docker/) - Monitor your GitHub Repos with Docker and Prometheus.
 - [Docker daemon metrics in Prometheus](https://medium.com/lucjuggery/docker-daemon-metrics-in-prometheus-7c359c7ff550) - Docker daemon metrics in Prometheus by Luc Juggery.
 - [Prometheus Monitoring Tutorial (10 minutes)](https://pagertree.com/2017/12/01/prometheus-tutorial/) - Simple tutorial to create a Prometheus monitoring stack with Grafana, AlertManager, and PagerTree by Austin Miller.
+- [Prometheus-Basics](https://github.com/yolossn/Prometheus-Basics) - Beginner Friendly Introduction to Prometheus
```
Comment (roaldnefs) [inline README.md]:
```suggestion
- [Prometheus-Basics](https://github.com/yolossn/Prometheus-Basics) - Beginner friendly introduction to Prometheus by [yolossn](https://github.com/yolossn).
```

**Review by roaldnefs — CHANGES_REQUESTED** [review]
Thanks @yolossn for you contribution. Let's get it merged after the suggestion has been applied.

**Comment by yolossn** [comment]
> Thanks @yolossn for you contribution. Let's get it merged after the suggestion has been applied.

Done 👍

**Review by roaldnefs — APPROVED** [review]
LGTM!

**Pull request merged**